index.html

```diff
@@ -268,7 +268,7 @@ <h2 data-i18n="dgc-getting-started" class="page-section-heading text-center text
                           <li><a class="dropdown-item select-lang" data-lang="de">Deutsch</a></li>
                           <li><a class="dropdown-item select-lang" data-lang="es">Español</a></li>
                           <li><a class="dropdown-item select-lang" data-lang="fr">Français</a></li>
-                          <li><a class="dropdown-item select-lang" data-lang="it">Italiano</a></li>
```

Comment:
Please undo this deletion :)

Reply:
Would appreciate if you fixed the indentation on the line again so it looks clean. Sorry for nitpicking.

Reply:
Done.
Sir It's okay! you can ask for any changes :)